llvm/test/DebugInfo/fixed-point.ll

```diff
@@ -0,0 +1,26 @@
+;; This fixes https://github.com/llvm/llvm-project/issues/81555
+;; Note this isn't correct dwarf but we just need to assert this doesn't crash.
+; RUN: %llc_dwarf %s -filetype=obj
```

Comment:
Can you still add CHECK lines to see what the expected output is?
That will make the test more robust against future changes. Otherwise this test passes even if llc_dwarf does nothing at all.

Reply:
Done